src/pages/home/report/ReportActionsView.js

```diff
@@ -196,7 +196,13 @@ class ReportActionsView extends React.Component {
     updateSortedReportActions(reportActions) {
         this.sortedReportActions = _.chain(reportActions)
             .sortBy('sequenceNumber')
-            .filter(action => action.actionName === 'ADDCOMMENT' || action.actionName === 'IOU')
+            .filter((action) => {
+                const message = _.last(lodashGet(action, 'message', null));
+                const html = lodashGet(message, 'html', '');
+                const text = lodashGet(message, 'text', '');
+                return ((html !== '' || text !== '') && action.actionName === 'ADDCOMMENT')
```

Comment:
NAB but I found this conditional slightly confusing. It's a bit subjective but I think this is clearer:

```javascript
return action.actionName === 'IOU'
    || (
        action.actionName === 'ADDCOMMENT'
        && !(html === '' && text === '')
    )
```